examples/org.eclipse.jface.text.examples/src/org/eclipse/jface/text/examples/codemining/CodeMiningDemo.java

```diff
@@ -55,6 +55,8 @@ public static void main(String[] args) throws Exception {
 		GridDataFactory.fillDefaults().grab(true, false).applyTo(endOfLineText);
 
 		ISourceViewer sourceViewer = new SourceViewer(shell, null, SWT.V_SCROLL | SWT.BORDER);
+		// test with line spacing
+		// sourceViewer.getTextWidget().setLineSpacing(20);
```

Comment:
```suggestion
		// {@code sourceViewer.getTextWidget().setLineSpacing(20);}
```

If this isn't meant as a code example, please remove it or uncomment it

Reply:
I have uncommented the line to test the behavior with line spacing enabled and indeed found bugs in the first run. I thought that it is helpful. I can also remove it if you like. Just tell me.

Reply:
Just for me to understand you correctly:

you get different bugs depending on if you enable or disable this line of code, yes?

In that case, I think that warrants a second test case (or maybe a parametrized test case?)